src/styles/_variables.scss

```diff
@@ -21,6 +22,7 @@ $c-light-grey: #6E7686;
     --c-dark-border: #{$c-river-styx};
     --c-grey: #{$c-river-styx};
     --c-light-grey: #{$c-light-grey};
+    --c-text-grey: #{$c-text-grey};
```

Comment:
unused?

Reply:
pulled from figma and used for `NavLink` color